eclipse-che/che-devfile-registry
# Support Devfile 2.3.0

### What does this PR do?
Add new `2.3.0` devfile schema.

### Screenshot/screencast of this PR



### What issues does this PR fix or reference?

https://github.com/eclipse-che/che/issues/22997

### How to test this PR?

1. Go to `tools/devworkspace-generator`
2. Execute `yarn && yarn build` to build the tool with the latest schema version.
3. Execute 
```
node lib/entrypoint.js --devfile-url:https://github.com/vinokurig/test.git --editor-entry:che-incubator/che-code/latest --output-file:devworkspace-che-code-latest.yaml
```
See: the yaml file is successfully generated from the `2.3.0` devfile.

### PR Checklist

[As the author of this Pull Request I made sure that:](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#pull-request-template-and-its-checklist)


### Reviewers

Reviewers, please comment how you tested the PR when approving it.
